README.md

```diff
@@ -380,6 +381,22 @@ df.write
   .save()
 ```
 
+## Transactional Guarantees
+
+This section describes `spark-redshift`'s transactional guarantees. For general information on Redshift's transactional guarantees, see the [Managing Concurrent Write Operations](https://docs.aws.amazon.com/redshift/latest/dg/c_Concurrent_writes.html) chapter in the Redshift documentation.
+
+**Appending to an existing table**: Writes use Redshift's [`COPY`](https://docs.aws.amazon.com/redshift/latest/dg/r_COPY.html) command, which is [atomic and transactional](https://docs.aws.amazon.com/redshift/latest/dg/copy-usage_notes-multiple-files.html). When loading data into Redshift, `spark-redshift` uses [manifests](https://docs.aws.amazon.com/redshift/latest/dg/loading-data-files-using-manifest.html) to guard against S3 eventual-consistency. Thus, `spark-redshift` appends to existing tables have the same atomic and transactional properties as regular Redshift `COPY` commands.
+
+**Overwriting an existing table**: By default, `spark-redshift` uses transactions to perform overwrites. Outside of a transaction, it will create an empty temporary table and append the new rows using a `COPY` statement. If the `COPY` succeeds, it will use a transaction to atomically delete the overwritten table and rename the temporary table to destination table.
```

Comment:
> Am I right that temp table creation and appending data to that using copy statement do not use transaction?

That's correct, although for simplicity / ease-of-reasoning I'm thinking that we should create the temp table inside of a transaction to ensure that it's deleted properly in case the load is interrupted. I'm wondering whether I should just do that cleanup ahead of the documentation patch here.